.github/workflows/ruby.yml

```diff
@@ -14,14 +14,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['2.7', '3.0', '3.1']
+        ruby: ['2.7', '3.0', '3.1', '3.2']
```

Comment:
Just a general note - I think we should drop support for 2.7 in line with Ruby EOL.

```
Ends in 2 months and 2 weeks(31 Mar 2023)
```